### PR TITLE
J

### PR DESCRIPTION
**Pull request by Abinash018**
this  is the pin related code that

### CODE REVIEW

**Comment by Abinash018** [comment]
pin

File: src/components/LoginForm.jsx [hunk 1]
```diff
@@ -19,6 +19,9 @@ const Modal = () => {
       localStorage.setItem('username', username);
```
Comment (saransh23e4) [inline src/components/LoginForm.jsx:19]: hope it works

**Review by saransh23e4 — APPROVED** [review]

File: src/components/LoginForm.jsx [hunk 2]
```diff
@@ -19,6 +19,9 @@ const Modal = () => {
...
       localStorage.setItem('password', password);
       localStorage.setItem('location', location);
+      localStorage.setItem('pin', pin);
+
```
Comment (Abinash018) [inline src/components/LoginForm.jsx:23]: i hope it works well

File: src/components/LoginForm.jsx [hunk 3]
```diff
@@ -19,6 +19,9 @@ const Modal = () => {
       localStorage.setItem('username', username);
       localStorage.setItem('password', password);
       localStorage.setItem('location', location);
+      localStorage.setItem('pin', pin);
```
Comment (Abinash018) [inline src/components/LoginForm.jsx:22]: **_this will be add pin in your local storage_**

File: src/components/LoginForm.jsx [hunk 4]
```diff
@@ -19,6 +19,9 @@ const Modal = () => {
       localStorage.setItem('username', username);
       localStorage.setItem('password', password);
       localStorage.setItem('location', location);
```
Comment (Abinash018) [inline src/components/LoginForm.jsx:21]: this will be add location

File: src/components/LoginForm.jsx (hunk 1 above)
Comment (Abinash018) [inline src/components/LoginForm.jsx:19]: this will be set the user name

**Review by Abinash018 — COMMENTED** [review]
pin location user name

**Pull request merged**